app/src/main/java/org/thoughtcrime/securesms/loki/activities/EditClosedGroupActivity.kt

```diff
@@ -37,8 +37,15 @@ import java.io.IOException
 
 class EditClosedGroupActivity : PassphraseRequiredActionBarActivity() {
     private val originalMembers = HashSet<String>()
+    private val zombies = HashSet<String>()
     private val members = HashSet<String>()
+    private val allMembers: Set<String>
+        get() {
+            return if (zombies.isNotEmpty()) (members + zombies)
```

Comment:
I need to double check, I initially did that one because I was getting an empty String at the end of the list (-> [ID1, ID2, ]) when zombies was empty, but I might have solved that somewhere else afterwards.

Reply:
Ah I see, but this check wouldn't even catch that actually (it'd just check that the list isn't empty, so if there's a `""` in the list the check would evaluate to `false`).

Reply:
Yeah, then I definitely forgot to clean that :)